src/components/Documentation.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+    <section class="section" id="dochome">
+    <div class="columns is-mobile is-centered">
+      <div class="column is-three-quarters-tablet is-half-desktop has-text-centered">
+        <div class="block">
+          <h1 class="title is-1">Documentation</h1>
+          <p>Resources for using Pangeo Forge</p>
+        </div>
+      </div>
+    </div>
+    <div class="content" id="firststeps">
+        <h3 class="title is-3">First steps</h3>
+        <p>New to Pangeo Forge?  Start here!</p>
+        <ul>
+            <li><strong>What is Pangeo Forge?</strong> - Read more about what Pangeo Forge is and
```

Comment:
I am open to hearing other opinions but at first pass I was thinking it could just point to https://github.com/pangeo-forge/roadmap.  The readme information specifically on the Inspiration, Vision, and one paragraph per technical concept I think is an appropriate level of information.

Reply:
In the latest commit I pointed this to https://pangeo-forge.readthedocs.io/en/latest/what_is_pangeo_forge.html.  The content that is there I think is nice.  I also imagine a technical description similar to [this](https://gist.github.com/rwegener2/b7f1b73fce11740f2836306f24bd6102#mid-level-technical-description) would be a good thing there.